### ISSUE TITLE
RawPedia down

### ISSUE THREAD

**Issue by Torsitano** [Beep6581/RawTherapee#7244]
**Short description**
I was trying to view the documentation when navigating from the main site, and I get a connection timeout. I tried on both my desktop and my phone(off WiFi).

**Steps to reproduce**
Navigate to https://rawpedia.rawtherapee.com/

**Expected behavior**
Documentation is available

**Additional information**
I saw people talking about improving the documentation in other issues, but no open issue about it being unavailable.

![image](https://github.com/user-attachments/assets/5685fdfc-09cb-459a-b8bd-5ad95b8a1e77)

**Comment by wilchak** [comment]
Yes, it's been down for a few weeks. This should be considered a high-priority issue since the instructions for building/compiling RT are hosted there.

**Comment by Lawrence37** [comment]
Additional information on the forum: https://discuss.pixls.us/t/is-rawpedia-down/46254

Until the site comes back up, you can use https://rawpedia.pixls.us/Main_Page. The "Main_Page" part is important, otherwise it will redirect to rawpedia.rawtherapee.com.

**Comment by Beep6581** [comment]
The Hungarian server that hosts RawPedia shows that we have no account on it. I am reaching out to Gabor who created the account to see what he knows about this, and to try and get a backup dump if still possible. Stay tuned.

**Comment by patdavid** [comment]
If I recall we had talked about migrating the wiki to a static site at one point - is that still a possibility?
I feel like the number of contributors to the wiki are relatively small and could be managed with access to a repo and writing pages in markdown possibly?

**Comment by patdavid** [comment]
> The Hungarian server that hosts RawPedia shows that we have no account on it. I am reaching out to Gabor who created the account to see what he knows about this, and to try and get a backup dump if still possible. Stay tuned.

Ok, the mediawiki instance is hosted on an old server of pixls.us at stablehost. The actual wiki is still up and running just fine (an alternate address you can reach it to see is: https://rawpedia.pixls.us/Main_Page).

The problem is the DNS entry I see for `rawpedia.rawtherapee.com` is to `184.154.46.85`.  
Rawpedia is actually located at `213.109.149.79`.

```bash
❯ ping rawpedia.rawtherapee.com
PING rawpedia.rawtherapee.com (184.154.46.85) 56(84) bytes of data.
```

It might be a good opportunity to look at migrating the docs to a static system (hugo, mdbook, gitbook, mkdocs, etc)?

**Comment by Lawrence37** [comment]
@patdavid We have a discussion about possibly replacing MediaWiki in RawTherapee/RawPedia#19.

**Comment by Entropy512** [comment]
Yup.  I haven't been around much (transitioning to new job, etc) but will hopefully have more time to resume contributions next year.

One thing that would help with experimentation of a rehost in a static generator would be a dump of the current content.  I've seen two examples of procedures for migrating from mediawiki to an SSG, which I mentioned long ago at https://github.com/Beep6581/RawTherapee/issues/6999#issue-2192880386

Specifically one of these:
https://gitlab.apertis.org/docs/mediawiki-to-hugo-conversion
or
https://github.com/xenlo/mediawiki-2-hugo

It looks like the Apertis approach would require less effort from the person with actual access to the mediawiki backend

**Comment by plantarum** [comment]
I understand transitioning to a new documentation system is not a small task. It also looks like the documentation available at pixls.us is not the latest version, as at least some of the pages there don't match what I see with the current RT 5.11.

Is there any way to get the existing current documentation back online while the larger issues are dealt with?

**Comment by Lawrence37** [comment]
@plantarum Based on Pat's comment, it seems the pixls.us one is the actual wiki. Even if it's not, it is fairly up-to-date. The last update before rawpedia.rawtherapee.com went down was just one month prior: http://rawpedia.pixls.us/Special:RecentChanges?hidebots=1&hidelog=1&limit=50&days=90&enhanced=1&urlversion=2

**Comment by Desmis** [comment]
@Lawrence37 @patdavid 

I am attaching a copy of what I just wrote on pixls.us
https://discuss.pixls.us/t/is-rawpedia-down/46254

I have no information on what is happening, except for the exchanges on the attached issue.
[Issue 7244](https://github.com/Beep6581/RawTherapee/issues/7244)

I am the last one to have made an update on November 10 (on the “copy” of Pixls.us, by tinkering and with impressive waiting times, therefore not usable - it was just the title of the documentation for GHS : Generalized Hyperbolic Stretch), but the one on the real site dates from September 24.

Documentation updates are pending. Of course it was planned to do something else, but it is not going to happen “by itself”… So what to do in the meantime?

Of course, within the limits of my abilities (I am not at all a specialist in servers and websites), I am ready to contribute if necessary.

Jacques

**Comment by Benitoite** [comment]
The editing accounts seem to not be copied, as I had to create a new account but with no editing rights.

**Comment by Beep6581** [comment]
I just regained access to the Hungarian RawTherapee server, but my joy was shortlived because it went down a few minutes later... Aside from server down time, the reason RawPedia hasn't been functioning seems to be related to the obsolete PHP version that MediaWiki uses.

When the server is back online I will make a data dump, and then over the holidays migrate RawPedia to some sort of static markdown solution. I want clear separation of content from styling. I want everyone with repo access to be able to edit markdown, and for the commit to update the user-facing documentation, so to automatically trigger a rebuild if necessary. Options include Hugo, MkDocs, Docusaurus, mdBook, GitBook. Shoot if you want to suggest something else.

And I want the new documentation host to be run by pixls.us so we don't have these issues in the future.

**Comment by ajaimenon** [comment]
This is so unfortunate, I have been reading the very useful documentations at rawpedia.pixls.us/ site as a user, knowing that the .com did not work and now .us is not available anymore as well.

**Comment by Lawrence37** [comment]
@patdavid Do you have any idea what happened to the rawpedia.pixls.us instance?

**Comment by plantarum** [comment]
I can contribute to transferring the documentation to the new system. I'm far from an expert in RawTherapee, but can contribute work in Markdown/Hugo formatting, updating screenshots, and perhaps some light text editing (ie clarity/grammar) in the process.

**Comment by purew** [comment]
FYI: Looks like the docs are available via archive.org at least. Last scan from Oct 16th:

https://web.archive.org/web/20241016233750/http://rawpedia.rawtherapee.com/Main_Page

EDIT: No, that was a short-lived joy. Seems they only archived the index page.

**Comment by Beep6581** [comment]
This is to clarify the confusion and misinformation spread here and in the forum:

RawPedia was initially hosted by me on the Hungarian server, but it has been kindly hosted by the `pixls.us` crew since before at least 2021.

[rawpedia.pixls.us](https://rawpedia.pixls.us/Main_Page) is not some outdated copy - it is in fact the official and only source.

For whatever reason, [rawpedia.rawtherapee.com](http://rawpedia.rawtherapee.com/) no longer points to the correct IP, while [rawpedia.pixls.us](https://rawpedia.pixls.us/) did. However, now even the latter does not work.

I never had access to the `pixls.us` server, so this is out of my hands.

I hope we are provided with a dump of the data so that we can host the documentation on a static host as soon as possible.

Ping @patdavid

**Comment by KarstenHahn** [comment]
Hi @Beep6581 and everyone,

as a Long-year User of RT it is time to give something back to the Community… 

I think that a temporary server would be useful until a migration to a new documentation system has been completed. I could provide this server. What do you think? I have never transported MediaWiki before, but it should work somehow. If you agree, then provide me with a dump of rawpedia and I'll try to install it. If it works, then we can change the DNS entry.

best greetings, Karsten

**Comment by dvdkon** [comment]
I [independently discovered](https://discuss.pixls.us/t/is-rawpedia-down/46254/38?u=dvdkon) the DNS problem, not knowing about this GH issue. I made a dump using [a fork of the wikiteam tools](https://github.com/mediawiki-client-tools/mediawiki-dump-generator.git), so someone interested should have no problem putting up a copy of the wiki.

**Comment by Beep6581** [comment]
@dvdkon could you share this dump with me Morgan Hardwood in the forum?

**Comment by dvdkon** [comment]
> @dvdkon could you share this dump with me Morgan Hardwood in the forum?

I've already posted it publically elsewhere, so [here's a link](https://dvdkon.ggu.cz/_/rawpediarawtherapeecom-20250102-wikidump.tar.gz). (I forgot to also post it here, sorry.)

**Comment by plantarum** [comment]
Not sure if this is helpful, but I found a mediawiki -> markdown converter and applied it to the dump from @dvdkon . There were a handful of Italian pages, and one Japanese page, that caused the conversion to fail. Not knowing anything about Italian, Japaenese, or Mediawiki, I just deleted these pages. The result is here:

https://github.com/plantarum/rawpedia_markdown/tree/main

I also pruned out several hundred empty files, each corresponding to an image file.

I'm happy to continue working on this if it's useful. David's data dump includes a directory with all the image files, and the markdown conversion includes (broken) links to the images. I think I can fix the links with a bit of scripting.

**Comment by plantarum** [comment]
I spent some time poking around with the dump yesterday. The export is generally ok, and doesn't take too much manual work to format as a github wiki. But for some reason the markdown converter didn't extract all of the pages, even though they appear to be present in the xml file. If I get time I will try out some other converters.

**Comment by hodgepig** [comment]
Hi everyone,

I may have missed it in this thread or the one on discuss.pixls.us, and I certainly don't mean this as a criticism in any way, but has anyone managed to recover the original/latest rawpedia and get it back online for everyone? I see that @KarstenHahn offered a server (excellent!) Like I imagine most rawtherapee users, I have neither the MediaWiki skills nor a server to help with this although I'm happy to help if I can.

A new project to migrate the documentation to something more modern is great but I think we need the existing/current documentation online first (if possible) otherwise users new and old are likely to move to Darktable or something else.

**Comment by Lawrence37** [comment]
As [stated by Pat](https://github.com/Beep6581/RawTherapee/issues/7244#issuecomment-2476725250), the server is still up. The issue is with the DNS resolution. One can override their DNS lookup for rawpedia.rawtherapee.com to `213.109.149.79` and access RawPedia just fine, demonstrating that it really is just a DNS problem. Someone needs to correct the IP address, but I don't know who can do that.

![Image](https://github.com/user-attachments/assets/6b47eccc-a628-48b1-9bfb-570d1f9626b4)

**Comment by Benitoite** [comment]
213.109.149.79 works but brings up a template for me.

**Comment by Lawrence37** [comment]
Simply entering the IP address will not work. The server pulls up the correct page using the host header. Browsers automatically take the hostname from the URL and sends it in the headers. By using the IP address in the address bar, the browser will send the IP address instead of "rawpedia.rawtherapee.com", so the server does not know you are looking for RawPedia.

**Comment by Benitoite** [comment]
Ok, I have to find out how to override the DNS then?  Can't anyone make the DNS update?

**Comment by Lawrence37** [comment]
Yes to your first question. On Linux, it's a simple as adding `213.109.149.79 rawpedia.rawtherapee.com` in `/etc/hosts`. No to your second question. It would be incredibly dangerous to allow anyone to change any DNS entry.

**Comment by plantarum** [comment]
> Ok, I have to find out how to override the DNS then? Can't anyone make the DNS update?

As I understand it, the owner of the domain is no longer associated with the project, or has been unavailable for the past months. They are the only pne who can update the DNS records.

**Comment by hodgepig** [comment]
This is not just a problem with rawpedia then. If the owner of rawtherapee.com is no longer involved or responding then the entire rawtherapee.com site could disappear too. Perhaps the development community should consider creating a new domain (rawtherapee.org?) with an active owner (or two) and move the entire site over so that 'we' have control over it again?

**Comment by KarstenHahn** [comment]
Hi all. The Domain rawtherapee.org is still available for 10$/year (Hover.com)… should I reserve it?Best greetings, KarstenVon meinem iPad gesendetAm 20.01.2025 um 13:46 schrieb hodgepig ***@***.***>:﻿
This is not just a problem with rawpedia then. If the owner of rawtherapee.com is no longer involved or responding then the entire rawtherapee.com site could disappear too. Perhaps the development community should consider creating a new domain (rawtherapee.org?) with an active owner (or two) and move the entire site over so that 'we' have control over it again?

**Comment by KarstenHahn** [comment]
Just bought  rawtherapee.org to save it for the project…
If you want, i will start a Domain Transfer for the owners of this GitHub Account here. Until that i will redirect all requess to rawtherapee.com.

Best greetings, Karsten



Von meinem iPad gesendetAm 20.01.2025 um 19:07 schrieb Karsten Hahn ***@***.***>:﻿Hi all. The Domain rawtherapee.org is still available for 10$/year (Hover.com)… should I reserve it?Best greetings, KarstenVon meinem iPad gesendetAm 20.01.2025 um 13:46 schrieb hodgepig ***@***.***>:﻿
This is not just a problem with rawpedia then. If the owner of rawtherapee.com is no longer involved or responding then the entire rawtherapee.com site could disappear too. Perhaps the development community should consider creating a new domain (rawtherapee.org?) with an active owner (or two) and move the entire site over so that 'we' have control over it again?

**Comment by ghorvath78** [comment]
Hi,
I'm Gabor Horvath, the owner of rawtherapee.com. I'm still paying for the domain and always will be. I just dont have the time to maintain the webpage. What shall I do to keep the webpage alive? I could give admin rights to any of you, and I can change the DNS address of rawpedia, too (the address 213.109.149.79 does not seem to work for me, though).

**Comment by ghorvath78** [comment]
If you find it easier to migrate everything to rawtherapee.org, I'm of course not against it. My priority is to keep the project alive,  and I really appreciate your effort. I can also redirect rawtherapee.com to rawtherapee.org, when it's up and running.

**Comment by KarstenHahn** [comment]
Hi @ghorvath78, thanks for your response.

no…. From my point of view it is better to correct the DNS entry because the address rawtherapee.com is well-known in the Internet.
Best greetings, Karsten

**Comment by Lawrence37** [comment]
I can still access RawPedia at 213.109.149.79. Also, the certificate for the https site expired on December 14.

**Comment by ghorvath78** [comment]
I modified the DNS entry, RawPedia is back to life.

**Comment by plantarum** [comment]
Wonderful, thank you so much Gabor!

I'm still happy to contribute however I can, to help take some of the burden off the core RT team. Whether that is writing/editing/converting docs, or whatever I could contribute to managing the website.

**Comment by Beep6581** [comment]
> As [stated by Pat](https://github.com/Beep6581/RawTherapee/issues/7244#issuecomment-2476725250), the server is still up. The issue is with the DNS resolution. One can override their DNS lookup for rawpedia.rawtherapee.com to `213.109.149.79` and access RawPedia just fine, demonstrating that it really is just a DNS problem. Someone needs to correct the IP address, but I don't know who can do that.
> 
> ![Image](https://github.com/user-attachments/assets/6b47eccc-a628-48b1-9bfb-570d1f9626b4)

@Lawrence37 how did you get that Headers tab?

**Comment by tanant** [comment]
@Beep6581 That looks like it's running in Firefox with the web developer tools open (`F12`, or `Ctrl + Shift + I` or `Alt menu > Tools > Browser Tools > Web Developer Tools` if that's what you were looking for, and same same in edge/chrome)

**Comment by Beep6581** [comment]
@tanant indeed, it's the Headers/Cookies/Requests/Response/Timings/Security bar that's missing on my end.

**Comment by tanant** [comment]
Oh wow, that's weird - I've never seen those missing. What do you see? This is hitting the site, and then on selecting one of the requests I get the view in the screenshot that you're referencing with the `Headers/Cookies/Requests/Response/Timings/Security` things

![Image](https://github.com/user-attachments/assets/a0427c18-159d-47b9-ac63-8ff62a9accfa)

If you can dump a screenshot with the 'network' devtool picked, and any old network request selected It might trigger off a memory as to what's missing and how to unhide it.

**Comment by Beep6581** [comment]
![Image](https://github.com/user-attachments/assets/01ffca3f-59dc-4c0a-8b52-962d4b847202)

In the words of the great Homer Simpson, "Doh". It appeared after I selected a line from the table and expanded the right panel.

![Image](https://github.com/user-attachments/assets/0155ef0e-18fe-45fa-963e-bde46222b780)
